src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -65,6 +65,10 @@ public function getOverrideListTagsSetting() {
    *   The event to process.
    */
   public function onRespond(ResponseEvent $event) {
+    // Don't try this if we're not in a Pantheon environment.
+    if (!function_exists('pantheon_clear_edge_paths')) {
```

Comment:
Is it harmful to have this method executing in non-Pantheon environments? For developers who need to troubleshoot which tags are getting returned, they probably want to do that in a local environment.